README.md

```diff
@@ -173,6 +173,12 @@ gatewayapi_gatewayclass_deleted{name="<GATEWAYCLASS_NAME>"} 1690879977
 gatewayapi_gatewayclass_status{name="<GATEWAYCLASS_NAME>",type="<Accepted>"} 1
 ```
 
+#### gatewayapi_gatewayclass_status_supported_features
+
+```
+gatewayapi_gatewayclass_status_supported_features{name="<GATEWAYCLASS_NAME>",type="<Accepted>",supportedFeatures=["HTTPRoute","HTTPRouteHostRewrite"]}
```

Comment:
```suggestion
gatewayapi_gatewayclass_status_supported_features{name="<GATEWAYCLASS_NAME>",type="<Accepted>",supportedFeatures=["<FeatureX>","<FeatureY>"]}
```
Listing 2 specific features gave me the impression that only 2 are possible in that metric label.

Also, maybe I missed this in the original PR, but labels should be snake case, so supported_features instead of supportedFeatures.
I'll have a closer look at the config & test as I may have overlooked it

Reply:
By the implementation, 64 features is the limit?